oidc_apis/utils.py

```diff
@@ -71,6 +72,12 @@ def after_userlogin_hook(request, user, client):
     except OidcClientOptions.DoesNotExist:
         pass
 
+    is_returning_from_idp = request.GET.get('first_authz', '') == 'false'
+    if not is_returning_from_idp and last_login_backend in settings.ALWAYS_REAUTHENTICATE_BACKENDS:
+        next_page = request.get_full_path()
+        return get_authorize_endpoint_redirect_to_login_response(
+            next_page, oidc_provider.settings.get('OIDC_LOGIN_URL'))
+
```

Comment:
Could there be a comment describing why this is done? Maybe in the doc that already describes the other things the function is doing.

Reply:
I rewrote the function documentation and added this new feature to it too.